README.md

```
@@ -167,7 +167,8 @@ You can define the next environment variables to change values from NGINX and PH
 | PHP8   | memory_limit            | 128M          | Maximum amount of memory in bytes that a script is allowed to allocate. This helps prevent poorly written scripts for eating up all available memory on a server. Note that to have no memory limit, set this directive to -1.                         |
 | PHP8   | post_max_size           | 8M            | Max size of post data allowed. This setting also affects file upload. To upload large files, this value must be larger than upload_max_filesize. Generally speaking, memory_limit should be larger than post_max_size.                                 |
 | PHP8   | upload_max_filesize     | 2M            | Maximum size of an uploaded file.                                                                                                                                                                                                                      |
-| PHP8   | zlib_output_compression | On            | Whether to transparently compress pages. If this option is set to "On" in php.ini or the Apache configuration, pages are compressed if the browser sends an "Accept-Encoding: gzip" or "deflate" header.                                               |
+| PHP8   | zlib_output_compression | On            | Whether to transparently compress pages. If this option is set to "On" in php.ini or the Apache configuration, pages are compressed if the browser sends an "Accept-Encoding: gzip" or "deflate" header.                                               |                                                                         |
```

Markdownlint

Expected: 4; Actual: 5; Too many cells, extra data will be missing
Table column count

(MD056, table-column-count)